src/state/actions/dispatchANTInteraction.ts

```diff
@@ -17,71 +16,107 @@ export default async function dispatchANTInteraction({
   workflowName: ANT_INTERACTION_TYPES;
   signer: ContractSigner;
   owner: string;
-  processId: ArweaveTransactionID;
+  processId: string;
```

Comment:
What's the reason for changing to string in all the places?

Reply:
The SDK uses a string so it makes the DX a bit smoother without this class